Debulleting the Porch - Guides pages (#557)
nephio-project/porch: docs/content/en/docs/4_tutorials_and_how-tos/working_with_package_revisions/inspecting-packages.md

porchctl rpkg get --namespace default
```

> This command queries Porch for all PackageRevisions in the specified namespace and displays a summary table with key information. It also shows PackageRevisions from all registered repositories.

Reviewer: ```suggestion
This command queries Porch for all PackageRevisions in the specified namespace and displays a summary table with key information. It shows PackageRevisions from all registered repositories in that namespace.
```

porchctl rpkg get porch-test.my-app.v1 --namespace default -o yaml
```

> This command retrieves the full PackageRevision resource and shows all metadata, spec, and status fields. Displayed in YAML format for easy reading.

Reviewer: ```suggestion
This command retrieves the full PackageRevision manifest, including all metadata, spec, and status fields. Displayed in YAML format for easy reading.
```

porchctl rpkg read porch-test.my-first-package.v1 --namespace default
```

> This command fetches PackageRevision resources and outputs to stdout and shows all KRM resources in ResourceList format. It displays the complete PackageRevision content.

Reviewer: ```suggestion
This command fetches PackageRevision resources and outputs to stdout, showing all KRM resources in ResourceList format. It displays the complete PackageRevision _content_.
```
